Source/VisualStudioTools/Private/VisualStudioToolsBlueprintBreakpointExtension.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <CoreMinimal.h>
+#include <EditorSubsystem.h>
+#include <EdGraph/EdGraph.h>
+#include <EdGraph/EdGraphNode.h>
+#include <EdGraph/EdGraphPin.h>
+#include <GraphEditorModule.h>
+#include <VisualStudioDTE.h>
+#include <Microsoft/COMPointer.h>
+#include "VisualStudioToolsBlueprintBreakpointExtension.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogUVisualStudioToolsBlueprintBreakpointExtension, Log, All);
+
+UCLASS()
+class UVisualStudioToolsBlueprintBreakpointExtension : public UEditorSubsystem
```

Comment:
should this also be behind the `#if ENGINE_VERSION >= 5` check? otherwise it would compile fine, but cause linking errors on lower versions

Reply:
just noticed a dummy implementation for older versions